### PR TITLE
Batch Explorer: added command to get compute node SSH settings

### PR DESCRIPTION
**Pull request by itowlson**
For your Linuxing pleasure!

### CODE REVIEW

File: CSharp/BatchExplorer/ViewModels/MainViewModel.cs [hunk 1]
```diff
@@ -1014,6 +1014,29 @@ public CommandBase OpenRDP
         }
 
         /// <summary>
+        /// Get SSH settngs for the selected ComputeNode
+        /// </summary>
+        public CommandBase GetSSH
```
Comment (matthchr) [inline CSharp/BatchExplorer/ViewModels/MainViewModel.cs]: Minor: Isn't the correct capitalization convention here `GetSsh`? I thought that acronyms were not all-caps unless they were only 2 characters long?

**Comment by matthchr** [comment]
Can you rebase up to master? I've fixed the travis build failure now so it should pass for you.

File: CSharp/BatchExplorer/ViewModels/MainViewModel.cs [hunk 2]
```diff
@@ -1014,6 +1014,29 @@ public CommandBase OpenRDP
...
+        {
+            get
+            {
+                return new CommandBase(
+                    (o) =>
+                    {
+                        var task = this.GetSSHSettingsAsync(this.SelectedComputeNode);
```
Comment (paselem) [inline CSharp/BatchExplorer/ViewModels/MainViewModel.cs]: Minor thing here, but may cause some confusion. When adding credentials to a linux node, a user can use ssh and/or a password... meansing ssh may not be there.

File: CSharp/BatchExplorer/ViewModels/MainViewModel.cs (hunk 1 above)
Comment (itowlson) [inline CSharp/BatchExplorer/ViewModels/MainViewModel.cs]: This was following our (inconsistent) capitalisation of RDP.  If memory serves, MainViewModel uses the capitalisation "RDP", and everywhere else uses the BCL-style "Rdp", and I followed that in each file.

**Pull request merged**